doc/src/index.md

```diff
@@ -1,4 +1,8 @@
-# Julia Documentation
+# Julia 0.7 Documentation
```

Comment:
Worth generating this from `VERSION`?

Reply:
I thought about it but felt it was an unnecessary complication. We could do it later if someone feels like it.